solr/core/src/java/org/apache/solr/servlet/SolrRequestParsers.java

```diff
@@ -86,7 +86,11 @@ public class SolrRequestParsers {
 
   private final HashMap<String, SolrRequestParser> parsers = new HashMap<>();
   private final boolean enableRemoteStreams;
+  public final boolean enableRemoteStreamsAtNodeLevel =
+      Boolean.parseBoolean(System.getProperty("solr.enableRemoteStreaming", "false"));
   private final boolean enableStreamBody;
+  public final boolean enableStreamBodyAtNodeLevel =
+      Boolean.parseBoolean(System.getProperty("solr.enableStreamBody", "true"));
```

Comment:
Do we need two different sysprops here? This `solr.enableStreamBody` is not documented in refGuide. Can't the same `solr.enableRemoteStreaming` be used for all remote streaming?